Documentation/deploy-master.md

```diff
@@ -74,9 +74,9 @@ After=flanneld.service
 
 [dropins]: https://coreos.com/os/docs/latest/using-systemd-drop-in-units.html
 
-#### Create the kubelet Unit
+### Create the kubelet Unit
 
-The kubelet is the agent on each machine that starts and stops Pods and other machine-level tasks. The kubelet communicates to the API server (also running on the master machine) with the TLS certificates we placed on disk earlier.
+The [kubelet](http://kubernetes.io/v1.0/docs/admin/kubelet.html) is the agent on each machine that starts and stops Pods and other machine-level tasks. The kubelet communicates with the API server (also running on the master machine(s)) with the TLS certificates we placed on disk earlier. On the master node, the kubelet is configured to communicate with the API server, but not register for cluster work, as shown in the `--register-node=false` line in the YAML excerpt below. This prevents user pods being scheduled on the master node(s), and ensures cluster work is routed only to task-specific worker nodes. Note that the kubelet running on a master node may therefore log repeated attempts to post status to the API server. These warnings are expected behavior and can be ignored. Future Kubernetes releases plan to [handle this common deployment consideration more gracefully](https://github.com/kubernetes/kubernetes/issues/14140#issuecomment-142126864).
```

Comment:
Should we just say "master nodes" (plural) everywhere and get rid of the `(s)`? It's going to be the main use-case and it doesn't hurt your understanding if you are only running one, IMO.